### PR TITLE
[OSCD] Create sysmon_abusing_debug_privilege.yml

### PR DESCRIPTION
**Pull request by semanurguneysu**
Task 74 from  #574

### CODE REVIEW

**Comment by aw350m33d** [comment]
Hello, @semanurguneysu! We are glad that you have joined the sprint!

Unfortunately, your rule has failed automatic validation tests. Please look at what happened there. Thank you.

**Comment by semanurguneysu** [comment]
> 
> 
> Hello, @semanurguneysu! We are glad that you have joined the sprint!
> 
> Unfortunately, your rule has failed automatic validation tests. Please look at what happened there. Thank you.

Hello, 
Fixed.Thank you.

**Comment by aw350m33d** [comment]
Hello Semanur! Sorry to keep you waiting so long. 

There are several questions:
1) Unfortunately, there is a broken link in `references` section. At least it didn't open for me, which means that other readers may have the same problems. Can you fix it?
2) We are currently trying to use `field modifiers` wherever possible. Could you replace the use of * with modifiers (`endswith` and `contains`)?
3) Please tell me the purpose of the `filter3`. Which command lines should be detected by your rule?

**Comment by semanurguneysu** [comment]
> Hello Semanur! Sorry to keep you waiting so long.
> 
> There are several questions:
> 
>     1. Unfortunately, there is a broken link in `references` section. At least it didn't open for me, which means that other readers may have the same problems. Can you fix it?
> 
>     2. We are currently trying to use `field modifiers` wherever possible. Could you replace the use of * with modifiers (`endswith` and `contains`)?
> 
>     3. Please tell me the purpose of the `filter3`. Which command lines should be detected by your rule?

Hello , 

1.I checked the link twice.It worked for me.Could you please check again 
https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-74-638.jpg

2.Replaced with modifiers.

3.It was logical error.I was trying to exclude filter3. Mentioned fields can generate noise due to event (For example adding value to routing table)

File: rules/windows/process_creation/sysmon_abusing_debug_privilege.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+title: Abused Debug Privilege by Arbitrary Parent Processes
+id: d522eca2-2973-4391-a3e0-ef0374321dae
+status: experimental
+description: Detection of unusual child processes by different system processes
+references:
+    - https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-74-638.jpg
+date: 2020/10/07
+tags:
+    - attack.privilege_escalation
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/sysmon_abusing_debug_privilege.yml:9]: Hello @semanurguneysu ! Could you please also add the ATT&CK technique to the tags? Thank you!

**Comment by aw350m33d** [comment]
Yes, the link from the comment works. Is it the same? ¯ \ _ (ツ) _ / ¯ 

A few more wishes. 

Could you change the names of the search IDs to more meaningful ones, or at least to something like `selection*`? Since the name `filter*` is usually used to exclude unnecessary events (just like you did with `filter3`).

Also, it might be worth including spaces in the `filter3` entries to pinpoint word boundaries. And if we want to search for the `route ADD` command, then we can add `all` field modifier. _Do you think that makes sense?_

And the last question. _Why did you use one backslash in the paths and two in the username?_ Usually, a single slash is used in the username (for example https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_whoami_as_system.yml), although the conversion result is the same for almost all backends, it seems that for QRadar the number of backslashes matters.
This is test example:
image

I do not work with this system, so it is difficult for me to judge whether the number of backslashes really matters. I wonder why you put two?

**Comment by aw350m33d** [comment]
Hey, Semanur! 😃 
What do you think about my message?

**Comment by semanurguneysu** [comment]
Hello ,

I updated my rule but I have problem about validation tests.I have seen same error for other pull requests too
.Is it a general issue? 

----
!!! Query check failed!
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fee4f62ffd0>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7fee4f668340>, 1010.045591554)]']
connector: <aiohttp.connector.TCPConnector object at 0x7fee4f62ffa0>
Makefile:115: recipe for target 'test-backend-es-qs' failed
make: *** [test-backend-es-qs] Error 3
Error: Process completed with exit code 2.

**Comment by aw350m33d** [comment]
Does this error appear if you run tests locally?

**Comment by yugoslavskiy** [comment]
Hello @semanurguneysu! I've checked your rule locally, it works fine.
Thank you very much for your contribution!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Comment by semanurguneysu** [comment]
> Hello @semanurguneysu! I've checked your rule locally, it works fine.
> Thank you very much for your contribution!

Thank you.

**Pull request merged**